core/src/main/java/org/mule/runtime/core/api/client/AbstractBaseOptionsBuilder.java

```diff
@@ -39,4 +47,9 @@ protected Long getResponseTimeout()
     {
         return responseTimeout;
     }
+
+    public boolean isOutbound()
```

Comment:
Since this is part of the API we should try not to expose methods that are required to work with transports.